packages/cli/package.json

```diff
@@ -31,18 +31,21 @@
     "lib"
   ],
   "keywords": [
+    "openzeppelin",
+    "openzeppelin-sdk",
+    "@openzeppelin/cli",
```

Comment:
I wouldn't include the full package name as a keyword

Reply:
I would add "ethereum", "solidity", and "smart-contracts" as keywords though (since we're at it)